pkg/machinepool/machinepool.go

```diff
@@ -1449,6 +1456,25 @@ func getMachinePoolReplicas(cmd *cobra.Command,
 			err = fmt.Errorf("Expected a valid number of replicas: %s", err)
 		}
 	}
+	if autoscaling && minReplicas <= 0 {
```

Comment:
Could these checks be put into a function to avoid code repetition and make it cleaner?